resources/views/themes/default1/order/index.blade.php

```diff
@@ -220,8 +220,10 @@
         $('#order-table').DataTable({
             processing: true,
             serverSide: true,
-             stateSave: false,
-            order: [[ 7, "desc" ]],
+            stateSave: false,
+            // if in request sort field is present, it will take that else default order
+            // need to stringify the sort_order, else it will be considered as a javascript variable
```

Comment:
What if we select version less than equal to and version greater than equal to at once. Should it be that if we select either of the one the other gets disabled or some validation of that kind?

Reply:
Similarly if we select paid products and version less than equal to (lets say 2.0.1), it should show the only the products that are paid and is less that v2.0.1. Just mentioning, don't know if it is tested or not.

Reply:
Do we need to remove the option for searching the exact version?

Reply:
> What if we select version less than equal to and version greater than equal to at once. Should it be that if we select either of the one the other gets disabled or some validation of that kind?

it is a range. It will work even if you select both

Reply:
> Similarly if we select paid products and version less than equal to (lets say 2.0.1), it should show the only the products that are paid and is less that v2.0.1. Just mentioning, don't know if it is tested or not.

yes, it will work that way

Reply:
> Do we need to remove the option for searching the exact version?

it can be acheived if you select both options as same. Though it has less use-cases, its not a good idea to give a seperate option for that

Reply:
> > Similarly if we select paid products and version less than equal to (lets say 2.0.1), it should show the only the products that are paid and is less that v2.0.1. Just mentioning, don't know if it is tested or not.
> 
> yes, it will work that way

It is showing unpaid products also even if we select paid products along with version less than dropdown